### ISSUE TITLE
Publish-NavContainerApp deletes the app file

### ISSUE THREAD

**Issue by marknitek** [microsoft/navcontainerhelper#492]
The current Version of Publish-NavContainerApp deletes the app file. 

I guess the following code is responsible for that:
```
    $containerAppFile = Get-NavContainerPath -containerName $containerName -path $appFile
    if ("$containerAppFile" -eq "") {
        $containerAppFile = Join-Path "c:\run\my" ([System.IO.Path]::GetFileName($appFile))
        Copy-FileToNavContainer -containerName $containerName -localPath $appFile -containerPath $containerAppFile
        $copied = $true
    }

...

    if ($copied) { 
        Remove-Item $appFile -Force
    }

```
I guess it should remove the ContainerAppFile instead?    
Or was the intention to remove the appFile if its downloaded from an url like here:
```
    $copied = $false
    if ($appFile.ToLower().StartsWith("http://") -or $appFile.ToLower().StartsWith("https://")) {
        $appUrl = $appFile
        $appFile = Join-Path $extensionsFolder "$containerName\my\$([System.Uri]::UnescapeDataString([System.IO.Path]::GetFileName($appUrl).split("?")[0]))"
        (New-Object System.Net.WebClient).DownloadFile($appUrl, $appFile)
        $copied = $true
    }
```

**Comment by freddydk** [comment]
You are right - the intention was to delete a downloaded or copied app

**Comment by freddydk** [comment]
Shipped in 0.6.1.3 (just made it before publishing:-))
